docker-compose.yml

```diff
@@ -8,13 +8,22 @@ services:
     container_name: zookeeper
     ports:
       - 2181:2181
+      - 2182:2182
     environment:
-      ZOOKEEPER_SERVER_ID: 1
       ZOOKEEPER_CLIENT_PORT: 2181
       ZOOKEEPER_TICK_TIME: 2000
+      ZOOKEEPER_SECURE_CLIENT_PORT: 2182
+      ZOOKEEPER_SERVER_CNXN_FACTORY: org.apache.zookeeper.server.NettyServerCnxnFactory
+      ZOOKEEPER_SSL_KEYSTORE_LOCATION: /etc/kafka/secrets/kafka.zookeeper.keystore.jks
+      ZOOKEEPER_SSL_KEYSTORE_PASSWORD: confluent
+      ZOOKEEPER_SSL_KEYSTORE_TYPE: PKCS12
```

Comment:
Is there a ticket to fix this discrepancy (filename extension is `.jks` but keystore type is `pkcs12`)?

Reply:
i'll file one to track